### ISSUE TITLE
[Bug]: Cancel Attachment

### ISSUE THREAD

**Issue by Beginner555989** [YimMenu/HorseMenu#200]
### Describe the bug

the troling things are soo nice but how we stop it ? bcz if u ride someone u cant stop u stuck till u leave the game or session

### Steps To Reproduce

player options .... troling slap or any other option .. you will stuck slaping the guy there is no stop option

### Expected Behavior

![Screenshot (112)](https://github.com/user-attachments/assets/93c15dc3-8d3e-450e-931d-36fe849d8645)

### Logs and/or screenshots

```
<paste your logs here>
```f


### Store

Steam

### Language

English

### Additional context

fffasda

### Build

- [X] I've confirmed the issue exists on the latest version of HorseMenu

**Comment by tuyilmaz** [comment]
Try using the "Stop Animations" option in Self->Animations

**Comment by Beginner555989** [comment]
> Try using the "Stop Animations" option in Self->Animations

it worked ;) tnx

**Comment by Beginner555989** [comment]
> Try using the "Stop Animations" option in Self->Animations

didnt work

**Comment by Rxann** [comment]
Working on a fix right now

**Comment by Rxann** [comment]
Closed by #210
